src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/BamlMapTable.cs

```diff
@@ -1509,7 +1509,9 @@ internal void LoadTypeInfoRecord(BamlTypeInfoRecord record)
                     Debug.Assert(targetType != null && memberName != null);
 
                     MemberInfo memberInfo = XamlTypeMapper.GetStaticMemberInfo(targetType, memberName, false);
-                    if (memberInfo is PropertyInfo)
+                    if (memberInfo is PropertyInfo &&
```

Comment:
I find that most WPF code files put && at the head of the code line instead of the end of it.

Reply:
I follow the style like line 450 in the same file.